### DISCUSSION TITLE
HTML formatting missing in error message on failed discount request

### DISCUSSION THREAD

**Discussion by FLAK-ZOSO** [community/community#129090] [GitHub Education]
### Hi everyone,

Nothing really interesting, <a href="lol exactly what I meant">here</a> is the little formatting error.
![image](https://github.com/community/community/assets/83119697/cf203033-65fa-41fc-a62d-8c37cc3d3972)

**Comment by Rishi-Sudhakar** [comment]
Haha XD , might be slower network which didn't render them as links, or a browser error, which just rendered the reference instead of the actual link.

**Comment by Tatendamarimo** [comment]
It looks like you need to fix some formatting issues
Please note, your request cannot be edited once it has been submitted, so please verify your details for accuracy before sending them to us. After clicking "Process my application," please wait as processing can take several seconds.

 Your application cannot be reviewed until you fix the following:

• Please secure your GitHub account with [two-factor authentication](https://github.com/settings/security). You may need to log out and log back in to GitHub before reapplying.

• Please complete your [GitHub billing information](https://github.com/settings/billing/payment_information) with your full name exactly as it appears in your academic affiliation document. You do not have to add a payment method. You may need to log out and log back in to GitHub before reapplying. If you have only a single legal name, enter it in both the first and last name fields.

• Please use your device camera to submit academic affiliation documents.

• Your image contains keywords indicating that it may be an acceptance letter. Only proof of current academic enrollment is acceptable.

• You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name exactly as it appears in your academic affiliation document. Please do not use a variation of your name or a nickname. Once you have updated your profile information, log out and log back into GitHub before reapplying.

Make sure to update any URLs and placeholders according to your actual application and needs.
